Mathlib/Topology/Defs/Basic.lean

```diff
@@ -98,6 +98,12 @@ class IsClosed (s : Set X) : Prop where
 def IsClopen (s : Set X) : Prop :=
   IsClosed s ∧ IsOpen s
 
+/--
+A set is locally closed if it is the intersection of some open set and some closed set.
+Also see `isLocallyClosed_tfae`.
```

Comment:
Can you mention the file `Mathlib/Topology/LocallyClosed.lean` where the API lemmas are? Thanks.

Reply:
```suggestion
Also see `isLocallyClosed_tfae` and other lemmas in `Mathlib/Topology/LocallyClosed`.
```